### PR TITLE
docs: migrate stories tsx

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/6086045892

### CODE REVIEW

File: packages/core/src/components/BreadcrumbsBar/__stories__/BreadcrumbsBar.stories.tsx [hunk 1]
```diff
@@ -32,7 +36,7 @@ export const Overview = {
   name: "Overview",
 
   args: {
-    items: [
+    children: [
```
Comment (talkor) [inline packages/core/src/components/BreadcrumbsBar/__stories__/BreadcrumbsBar.stories.tsx:39]: Using a different control is misleading... Should have been `children`

File: packages/core/src/components/Combobox/__stories__/Combobox.stories.tsx [hunk 2]
```diff
@@ -237,7 +237,12 @@ export const WithCategories = {
         justify={Flex.justify.START}
         align={Flex.align.START}
       >
-        <StoryDescription description="Regular" vertical align={Flex.align.START}>
+        <StoryDescription
+          description="Regular"
+          vertical
+          // @ts-ignore
```
Comment (talkor) [inline packages/core/src/components/Combobox/__stories__/Combobox.stories.tsx:243]: Will resolve it later in storybook-blocks, the components are not typed so it's quite a mess now

File: packages/core/src/components/Dialog/__stories__/Dialog.stories.tsx [hunk 3]
```diff
@@ -538,14 +545,23 @@ export const ControlledDialog = {
         content={
           <DialogContentContainer>
             <DialogContentContainer>
-              <Button kind={Button.kinds.SECONDARY} onClick={() => setIsOpen(false)}>
+              <Button
+                kind={Button.kinds.SECONDARY}
+                // @ts-ignore
```
Comment (talkor) [inline packages/core/src/components/Dialog/__stories__/Dialog.stories.tsx:550]: Something is wrong with the useSwitch hook, it's also not typed so it's a bit tricky now

File: packages/core/src/components/Modal/ModalHeader/ModalHeader.tsx [hunk 4]
```diff
@@ -70,6 +62,18 @@ export interface ModalHeaderProps extends VibeComponentProps {
...
 
+interface ModalHeaderWithOnlyTitle extends BaseModalHeaderProps {
+  title: ElementContent;
+  children?: never;
+}
+
+interface ModalHeaderWithOnlyChildren extends BaseModalHeaderProps {
+  title?: never;
+  children: ElementContent;
+}
```
Comment (talkor) [inline packages/core/src/components/Modal/ModalHeader/ModalHeader.tsx:73]: For some reason the `title` prop was mandatory even when `children` were provided, it came up in the story so I changed it. Now either of them is mandatory

File: packages/core/src/components/Modal/__stories__/Modal.stories.helpers.tsx [hunk 5]
```diff
@@ -16,7 +16,7 @@ export const useHelperOpenModalButton = ({
   openModalButtonRef,
   color = undefined,
   testId = undefined
-}) => {
+}: any) => {
```
Comment (talkor) [inline packages/core/src/components/Modal/__stories__/Modal.stories.helpers.tsx:19]: Some props are not provided by the component, will investigate later

File: packages/core/src/components/Slider/__stories__/Slider.stories.tsx [hunk 6]
```diff
@@ -84,8 +84,21 @@ export const WithLabels = {
...
-      <Slider prefix={{ icon: Video }} postfix={{ icon: Sound }} defaultValue={45} size={Slider.sizes.MEDIUM} />
+      <Slider
+        // @ts-ignore
+        prefix={{ icon: Sound }}
+        indicateSelection={true}
+        defaultValue={70}
+        size={Slider.sizes.SMALL}
+      />
+      <Slider
+        // @ts-ignore
```
Comment (talkor) [inline packages/core/src/components/Slider/__stories__/Slider.stories.tsx:95]: Something here is wrong with the icon type, will investigate later

File: packages/core/src/components/Steps/__stories__/Steps.stories.tsx [hunk 7]
```diff
@@ -157,9 +157,9 @@ export const StepsInsideATipseen = {
         <Tipseen
           position={Tipseen.positions.LEFT}
           modifiers={modifiers}
+          animationType={Tipseen.animationTypes.OPACITY_AND_SLIDE}
```
Comment (talkor) [inline packages/core/src/components/Steps/__stories__/Steps.stories.tsx:160]: Should have been here all along

File: packages/core/src/components/Tipseen/__stories__/Tipseen.stories.tsx [hunk 8]
```diff
@@ -27,15 +27,21 @@ export default {
...
+}: TipseenProps & TipseenContentProps) => {
   return (
     <Tipseen
       // The modifier's purpose is to prevent the tipseen from being displayed when the user scrolls the story upwards / downwards.
       // Therefore, there is no need to move this prop in your implementations.
       modifiers={modifiers}
       position={position}
       content={
-        <TipseenContent isDismissHidden={isDismissHidden} title={title}>
+        <TipseenContent hideDismiss={hideDismiss} title={title}>
```
Comment (talkor) [inline packages/core/src/components/Tipseen/__stories__/Tipseen.stories.tsx:44]: `isDismissHidden` is deprecated

**Review by YossiSaadi — APPROVED** [review]

**Pull request merged**